src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -241,6 +241,10 @@ private void move(Vector spdVec, double rot) {
    * @param rateLimit whether or not to use slew rate limiting
    */
   private void move(Vector spdVec, double rot, boolean rateLimit) {
+    Vector spdCommanded = spdVec;
+
+    spdVec.rot(0);
```

Comment:
What does this do? Why is it relevant?

Reply:
apparently things break it .rot() isn't called, which doesn't seem right to me, and I'll try to debug it in my free time, but this was needed according to testing

Reply:
According to Garrett, this will rotate it by 0 degrees. What did you encounter during testing leading to this being added?